pkg/yurthub/cachemanager/cache_manager_test.go

```diff
@@ -1098,7 +1115,7 @@ func TestCacheListResponse(t *testing.T) {
 			path:       "/api/v1/node",
 			resource:   "nodes",
 			namespaced: false,
-			cacheErr:   storage.ErrStorageNotFound,
+			queryErr:   storage.ErrStorageNotFound,
```

Comment:
I think we only need `expectResult` to verify the return values, so please delete `cacheErr` or `queryErr` field.

Reply:
The code has been updated, please review.